buildscripts/buildvue.js

```diff
@@ -94,11 +94,11 @@ function compileComponent(file){
  */
 function updateIndexFiles() {
     var jsFiles = readDirs(distBasePath+'/', distBasePath + '/js', '.js')
-    jsFiles.unshift('<!-- auto generated by build -->')
+    jsFiles.unshift('# auto generated by build')
     fs.writeFileSync(distBasePath+'/js.txt', jsFiles.join('\n'))
 
     var cssFiles = readDirs(distBasePath+'/', distBasePath + '/css', '.css')
-    cssFiles.unshift('<!-- auto generated by build -->')
+    cssFiles.unshift('# auto generated by build')
```

Comment:
i guess this is just personal preference?

Reply:
It's actually the comment style for `css.txt` in the client libs of the system